openedx/core/djangoapps/content_libraries/serializers.py

```diff
@@ -34,7 +36,7 @@ class ContentLibraryMetadataSerializer(serializers.Serializer):
     org = serializers.SlugField(source="key.org")
     slug = serializers.CharField(source="key.slug", validators=(validate_unicode_slug, ))
     bundle_uuid = serializers.UUIDField(format='hex_verbose', read_only=True)
-    collection_uuid = serializers.UUIDField(format='hex_verbose', write_only=True)
+    #collection_uuid = serializers.UUIDField(format='hex_verbose', write_only=True)
```

Comment:
@ChrisChV @bradenmacdonald: Did you mean to leave a commented out line of code? Shouldn't it just be deleted?

I only happened to look at this PR because the tests failed on master (at first) due to some issue uploading an artifact, but it passed after I re-ran (or maybe after Braden had)?

Reply:
Sorry, just an oversight on my part. I should have asked for that line to be deleted during the review. We're doing lots of work in this area though, so we'll remove it in a follow-up pass.